web/fonts/SourceSansVariable-Roman_LICENSE.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2010, 2012, 2014 Adobe Systems Incorporated (http://www.adobe.com/), with Reserved Font Name 'Source'. All Rights Reserved. Source is a trademark of Adobe Systems Incorporated in the United States and/or other countries.

```

Comment:
not sure it makes sense we have the license in the repos, but I thought it might make sense